snooty/parser.py

```diff
@@ -940,6 +944,21 @@ def validate_and_add_asset(
                 CannotOpenFile(Path(image_argument), err.strerror, line)
             )
 
+    def validate_relative_url(self, url_argument: str, line: int) -> None:
+        """Validate relative URL points to page within current docs site.
+        URLs can be of the form /foo, foo, /foo/"""
```

Comment:
Random thought, no change needed: I think ideally, users will use `/foo` since I believe that's the typical convention used for referring to other file names for other directives and roles. However, it does make sense to support the other forms as well since they're valid for navigation on the frontend